tests/foreman/endtoend/test_cli_endtoend.py

```diff
@@ -0,0 +1,406 @@
+    FAKE_0_PUPPET_REPO,
+    GOOGLE_CHROME_REPO,
+    PRDS,
+    REPOS,
+    REPOSET,
+)
+from robottelo.config import settings
+from robottelo.decorators import bz_bug_is_open
+from robottelo.test import CLITestCase
+
+from .utils import ClientProvisioningMixin
+# (too many public methods) pylint: disable=R0904
+
+
+class EndToEndTestCase(CLITestCase, ClientProvisioningMixin):
+    """End-to-end tests using the ``CLI`` path."""
+
+    def test_positive_find_default_org(self):
+        """Check if 'Default Organization' is present
+
+        @Feature: Smoke Test
+
+        @Assert: 'Default Organization' is found
+        """
+        result = Org.info({u'name': DEFAULT_ORG})
+        self.assertEqual(result['name'], DEFAULT_ORG)
+
+    def test_positive_find_default_loc(self):
+        """Check if 'Default Location' is present
+
+        @Feature: Smoke Test
+
+        @Assert: 'Default Location' is found
+        """
+        result = Location.info({u'name': DEFAULT_LOC})
+        self.assertEqual(result['name'], DEFAULT_LOC)
+
+    def test_positive_find_admin_user(self):
+        """Check if Admin User is present
+
+        @Feature: Smoke Test
+
+        @Assert: Admin User is found and has Admin role
+        """
+        result = User.info({u'login': u'admin'})
+        self.assertEqual(result['login'], 'admin')
+        self.assertEqual(result['admin'], 'yes')
+
+    def test_positive_end_to_end(self):
+        """Perform end to end smoke tests using RH and custom repos.
+
+        1. Create a new user with admin permissions
+        2. Using the new user from above
+            1. Create a new organization
+            2. Clone and upload manifest
+            3. Create a new lifecycle environment
+            4. Create a custom product
+            5. Create a custom YUM repository
+            6. Create a custom PUPPET repository
+            7. Enable a RedHat repository
+            8. Synchronize the three repositories
+            9. Create a new content view
+            10. Associate the three repositories to new content view
+            11. Publish content view
+            12. Promote content view to the lifecycle environment
+            13. Create a new activation key
+            14. Add the products to the activation key
+            15. Create a new libvirt compute resource
+            16. Create a new subnet
+            17. Create a new domain
+            18. Create a new hostgroup and associate previous entities to it
+            19. Provision a client
+
+        @Feature: End to End Test
+
+        @Assert: All tests should succeed and Content should be successfully
+        fetched by client.
+        """
+        # step 1: Create a new user with admin permissions
+        password = gen_alphanumeric()
+        user = make_user({u'admin': u'true', u'password': password})
+        user['password'] = password
+
+        # step 2.1: Create a new organization
+        org = self._create(
```

Comment:
Can't it be one line statement?
`org = self._create(user, Org, {u'name': gen_alphanumeric()})`